### DISCUSSION TITLE
return code 130 when pressing ESC

### DISCUSSION THREAD

**Discussion by rodhash** [junegunn/fzf#3308] [General]
When I run anything like "ls | fzf" and I just want to  quit FZF, pressing ESC quits with return code 130.

Can I change that with some keybinding / command to quit with RC 0?

**Comment by LangLangBart** [comment]
```sh
man fzf
# ...
# EXIT STATUS
#       0      Normal exit
#       1      No match
#       2      Error
#       130    Interrupted with CTRL-C or ESC
```

---
### become
- ref: [release 38](https://github.com/junegunn/fzf/releases/tag/0.38.0)
- ref: [fzf#turning-into-a-different-process](https://github.com/junegunn/fzf#turning-into-a-different-process)

```sh
ls | fzf --bind 'esc:become:'
```

**Comment by junegunn** [comment]
```sh
ls | fzf || :
```

**Reply by Delapouite** [reply]
Explanations about the non-obvious meaning of `:` in this snippet https://stackoverflow.com/questions/3224878/what-is-the-purpose-of-the-colon-gnu-bash-builtin
